dds/DCPS/ContentFilteredTopicImpl.cpp

```diff
@@ -30,7 +30,14 @@ ContentFilteredTopicImpl::ContentFilteredTopicImpl(const char* name,
   , filter_eval_(filter_expression, false /*allowOrderBy*/)
   , expression_parameters_(expression_parameters)
   , related_topic_(DDS::Topic::_duplicate(related_topic))
-{}
+{
+  if (DCPS_debug_level > 5) {
+      ACE_DEBUG((LM_DEBUG,
+                ACE_TEXT("(%P|%t) ContentFilteredTopicImpl::ContentFilteredTopicImpl() - ")
+                ACE_TEXT("Creating cft with filter <%C> which requires <%d> parameters\n"),
+                filter_expression, filter_eval_.number_parameters ()));
+  }
+}
```

Comment:
In this constructor the size of the expression_parameters is not checked using the same logic of set_expression_parameters.